bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifier.scala

```diff
@@ -75,7 +59,9 @@ trait BagVerifier[BagContext <: BagVerifyContext[BagPrefix], BagLocation <: Loca
     bag: Bag
   ): Either[BagVerifierError, Unit]
 
-  def createPrefix(path: String): BagPrefix
+  // What bucket is storing the "primary" copy of a bag?  This is the bucket
+  // that should be referred to in the fetch.txt URIs.
+  val bucket: String
```

Comment:
nit: "primaryBucket"? So we don't get it confused with other buckets we might be using.